### DISCUSSION TITLE
create a web GUI

### DISCUSSION THREAD

**Discussion by kanarelo** [alexmyczko/ruptime#17] [Ideas]
The application will need a gui and api for monitoring purposes.

We can discuss the specific objects / data items to track and how to track

![vetmu1xh5khc1](https://github.com/alexmyczko/ruptime/assets/393261/7890e2de-6f1e-4a98-a01c-f918a6ccef96)

**Comment by alexmyczko** [comment]
I have something very simple here now:

https://github.com/alexmyczko/ruptime/tree/debian/web

it's missing icons like /System/Library/CoreServices/CoreTypes.bundle/Contents/Resources
and a per host, and per site info still but it's a start.

also ACL is missing.

**Comment by kanarelo** [comment]
Nice, I will take a look today.

**Comment by alexmyczko** [comment]
already looks good to me, these are missing:

per host display
stats
sortability
network/host discovery
autogrouping (see milestones)
DONE one minute autorefresh

**Comment by kanarelo** [comment]
I think we will need more to give more information about the web gui. 
- How to set it up and/or how to access if it's already available in the server version. 
- Does it use a webserver? 
- How do we deploy it? 
- And how to access it for testing/development purposes.

**Comment by alexmyczko** [comment]
You need a webserver of your choice, one that can execute CGI.
